### PR TITLE
feat(card): add images of card

### PR DESCRIPTION
**Pull request by GusBaamonde**
- add styles of the card component
- add the html from card
- add all card states
- adjust the tooltip

See more: https://cdn.fromdoppler.com/doppler-ui-library/build920/cards-component.html

cards-states

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/scss/modules/_card.scss [hunk 1]
```diff
@@ -146,3 +172,144 @@
...
+
+.dpsg-tit-card {
+  font-size: 12px;
+  width: 285px;
+  text-transform: uppercase;
+}
+
+.dp-maximum {
+  border-radius: 3px;
+  background-color: #f2f2f2;
```
Comment (cbernat) [inline assets/scss/modules/_card.scss:220]: Color missing here.

**Comment by cbernat** [comment]
Some images are not showing correctly 
image

https://cdn.fromdoppler.com/doppler-ui-library/build919/cards-component.html

**Comment by cbernat** [comment]
Why is this a ul?

image

I think that without a list would be ok, we don't have more than one price by card right?

**Comment by cbernat** [comment]
The layout is looking very nice! 💪 

image

**Comment by GusBaamonde** [comment]
https://cdn.fromdoppler.com/doppler-ui-library/build920/cards-component.html

**Comment by GusBaamonde** [comment]
> Some images are not showing correctly
> image
> 
> https://cdn.fromdoppler.com/doppler-ui-library/build919/cards-component.html
> 
> Also I don't see the tooltips working 🤔 👀

Resolved! 👍

**Comment by GusBaamonde** [comment]
> Why is this a ul?
> 
> image
> 
> I think that without a list would be ok, we don't have more than one price by card right?

Due to an alignment problem.
The number can grow or the words and everything has to be aligned with the center.
An "ul" solves the whole lineup for us without weird things

**Comment by mdirago** [comment]
Excellent work Gus!

**Pull request merged**